pkg/kvstore/etcd.go

```diff
@@ -1162,7 +1214,6 @@ func (e *etcdClient) determineEndpointStatus(ctx context.Context, endpointAddres
 
 	e.getLogger().Debugf("Checking status to etcd endpoint %s", endpointAddress)
 
-	e.limiter.Wait(ctxTimeout)
```

Comment:
I've dropped the rate limiter wait here, to prevent that the status check fails due to rate limiting, as that would put a high pressure on etcd in case the connection gets restarted.